Geometry_Engine/Compute/FitLine.cs

```diff
@@ -35,82 +35,99 @@ public static partial class Compute
 
         public static Line FitLine(this IEnumerable<Point> points, double tolerance = Tolerance.Distance)
```

Comment:
worth adding a description here where we can explain the methodologies used, otherwise we are relying on users to look at the code (which I doubt many do).